### PR TITLE
Scala versions

### PR DESCRIPTION
**Pull request by ignasi35**
Supersedes #227

### CODE REVIEW

**Comment by johanandren** [comment]
Build fail is formatting of sbt files

**Comment by ignasi35** [comment]
Ok, seems we're finally getting to the meat of it:

```
[error] /home/travis/build/akka/akka-persistence-couchbase/core/src/main/scala/akka/persistence/couchbase/internal/CouchbaseSchema.scala:327:60: object JavaConverters in package collection is deprecated (since 2.13.0): Use `scala.jdk.CollectionConverters` instead
```

File: .travis.yml [hunk 1]
```diff
@@ -58,7 +58,7 @@ jobs:
     - env:
       - JDK="adopt@~1.11-0"
       - PRE_CMD="docker-compose -f docker/couchbase-1-node/docker-compose.yml up -d couchbase"
-      - CMD=";++2.12.8 test" # note that JDK 11 + Java requires Scala 2.12 because of extension getter ambiguity
+      - CMD=";++2.12.10 test" # note that JDK 11 + Java requires Scala 2.12 because of extension getter ambiguity
```
Comment (ennru) [inline .travis.yml]: This must be in sync with sbt: 2.12.11

Comment (ignasi35) [reply .travis.yml]: Good catch!

Comment (ignasi35) [reply .travis.yml]:
```suggestion
      - CMD=";++2.12.11 test" # note that JDK 11 + Java requires Scala 2.12 because of extension getter ambiguity
```

**Pull request merged**